src/EnergyPlus/DataSurfaces.cc

```diff
@@ -88,15 +87,6 @@ using namespace WindowManager;
 
 Array1D_string const cExtBoundCondition({-6, 0}, {"KivaFoundation", "FCGround", "OSCM", "OSC", "OSC", "Ground", "ExternalEnvironment"});
 
-// Parameters to indicate surface classes
```

Comment:
Removed some comments that were left behind at some point when the parameters were moved to an `enum`.